data/game/functions/managers/shulkers/place.mcfunction

```diff
@@ -1,8 +1,8 @@
 # Summon the shulker only if it's on the right island
 scoreboard players set #success var 0
-execute if score #shulker_type var matches 1 store success score #success var if predicate game:locations/red_island run function game:resources/prefabs/shulkers/variants/red
-execute if score #shulker_type var matches 2 store success score #success var if predicate game:locations/blue_island run function game:resources/prefabs/shulkers/variants/blue
-execute if score #shulker_type var matches 3 store success score #success var if predicate game:locations/green_island run function game:resources/prefabs/shulkers/variants/green
+execute if score #shulker_type var matches 1 store success score #success var if predicate game:locations/blue_island run function game:resources/prefabs/shulkers/variants/blue
+execute if score #shulker_type var matches 2 store success score #success var if predicate game:locations/green_island run function game:resources/prefabs/shulkers/variants/green
+execute if score #shulker_type var matches 3 store success score #success var if predicate game:locations/red_island run function game:resources/prefabs/shulkers/variants/red
 execute if score #shulker_type var matches 4 store success score #success var if predicate game:locations/yellow_island run function game:resources/prefabs/shulkers/variants/yellow
```

Comment:
Maybe i'm completly crazy but i prefer keeping the other order red 1 blue 2 green 3 and yellow 4
That's cause everything else is kind of organized this way. What I like about it is that we don't have to check for the id when working on it, it's by length ^^

Reply:
Yeah ik in alphabetical order but I think I still prefer by length that's why I'm saying I'm super weird...